src/scss/index.scss

```diff
@@ -1,117 +1,24 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:400|Lato:400');
-
-html {
-  font-size: 15px;
-}
-
-body {
-  background-color: white;
-  font-family: 'Lato', sans-serif;
-  font-weight: 400;
-  line-height: 1.65;
-  color: #333;
-  padding-top: 80px;
-}
-
-p {
-  margin-bottom: 1.15rem;
-
-   /* section header in docstring pages */
-  &.rubric {
-    border-bottom: 1px solid rgb(201, 201, 201);
-  }
-}
-
-a {
-  color: #005b81;
-  text-decoration: none;
-
-  &:hover {
-    color: #e32e00;
-    text-decoration: underline;
-  }
-}
-
-h1,
-h2,
-h3,
-h4,
-h5 {
-  margin: 2.75rem 0 1.05rem;
-  font-family: 'Open Sans', sans-serif;
-  font-weight: 400;
-  line-height: 1.15;
-}
-
-// offsetting html anchor titles to adjust for fixed header
-h2::before,
-h3::before,
-h4::before,
-h5::before,
-h6::before {
-  display: block;
-  content: '';
-  height: 80px;
-  margin: -80px 0 0;
-}
-
-h1 {
-  margin-top: 0;
-  font-size: 2.488em;
-  color: #130654;
-}
-
-h2 {
-  font-size: 2.074em;
-  color: #130654;
-}
-
-h3 {
-  font-size: 1.728em;
-}
+// override bootstrap variables
+$spacer: 1rem;
 
-h4 {
-  font-size: 1.44em;
-}
+// include core Bootstrap
+@import '../../node_modules/bootstrap/scss/bootstrap';
```

Comment:
This includes all of bootstrap css in our index.css, right? 
I don't know if this is best to solve https://github.com/pandas-dev/pydata-sphinx-theme/issues/89, but a downside of this is that on the actual site, this makes it difficult to see which parts of css is coming from main bootstrap, and which are customizations from this theme.

Reply:
Couldn't we just bundle bootstrap, minified, in this theme, and directly link it? I'd be +1 on that, I actually need to do this in a downstream theme anyway in order to get PDF printing to work

Reply:
You are correct, this includes all of bootstrap in our index.css. Bundling bootstrap as in this theme would make us lose the [configuration options](https://getbootstrap.com/docs/4.0/getting-started/webpack/) Bootstrap provides. 

A solution to might satisfy both would be to have 2 css outputs defined in Webpack.
1. Full bootstrap with custom configuration applied.
2. Custom css applied after bootstrap.

Would this be a solution for @jorisvandenbossche and @choldgraf  cases?

This should be a different PR, since it might be more complicated then it seems. Especially in combination with https://github.com/pandas-dev/pydata-sphinx-theme/issues/139 to allow bootstrap Mixins to be available in the custom css without bundling twice.

Reply:
I actually found a workaround on my end that didn't require bundling bootstrap, though I think there's still a benefit to this for working in an offline context. Depending on how complex it makes things I am interested!

Reply:
I think that at least sounds an interesting option to explore, to see if that would be possible to have two css outputs. 
But indeed fine to do as a follow-up